app/forms/otp_verification_form.rb

```diff
@@ -0,0 +1,22 @@
+class OtpVerificationForm
+  def initialize(user, code)
+    @user = user
+    @code = code
+  end
+
+  def submit
+    @success = user.authenticate_direct_otp(code)
+
+    result
+  end
+
+  private
+
+  attr_reader :code, :user, :success
+
+  def result
+    {
+      success?: success
```

Comment:
I am planning on adding more properties here in another PR, to track things like whether the user is setting up their phone for the first time or updating their existing one.